src/accordion/accordion.spec.ts

```diff
@@ -852,11 +862,122 @@ describe('ngb-accordion', () => {
 
       accordion.collapseAll();
       fixture.detectChanges();
+      fixture.detectChanges();
       expectOpenPanels(nativeElement, [false, false]);
     });
   });
 });
 
+if (isBrowserVisible('ngb-accordion animations')) {
+  describe('ngb-accordion animations', () => {
+
+    @Component({
+      template: `
+      <ngb-accordion activeIds="first" (ngbCollapseChange)="onCollapseChange()">
```

Comment:
1. Please add `$event` and check the values that are actually sent by the emitter
2. Please check that `(ngbCollapseChange)` is emitted with correct values in "normal" tests without animations